requirements-test.txt

```diff
@@ -0,0 +1,6 @@
+coverage==5.5
```

Comment:
You probably need to update CI so it properly installs this requirements for running tests

Reply:
He already did - the tests fail for a different reason, the same fail message is in the other PR which doesn't touch requirements